lib/prop/interval_strategy.rb

```diff
@@ -14,10 +14,20 @@ def counter(cache_key, options)
       end
 
       def increment(cache_key, options)
-        increment = options.fetch(:increment, 1)
-        raise ArgumentError, "Increment must be a Fixnum, was #{increment.class}" unless increment.is_a?(Fixnum)
+        change(cache_key, options.fetch(:increment, 1))
+      end
+
+      def decrement(cache_key, options)
+        if counter(cache_key, options) <= zero_counter
```

Comment:
hmm don't like this ... adds an extra lookup ... decrementing blow 0 is not that bad ... :D